plot_pixel_values.py

```diff
@@ -22,41 +24,103 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
```

Comment:
I don't think it hurts much in code complexity to leave that plot available
via an option.​